Gems/Multiplayer/Code/Include/Multiplayer/Components/NetworkHierarchyChildComponent.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <Multiplayer/Components/NetworkHierarchyBus.h>
+#include <Source/AutoGen/NetworkHierarchyChildComponent.AutoComponent.h>
+
+namespace Multiplayer
+{
+    class NetworkHierarchyRootComponent;
+
+    //! @class NetworkHierarchyChildComponent
+    //! @brief Component that declares network dependency on the parent of this entity
+    /*
+     * The parent of this entity should have @NetworkHierarchyChildComponent (or @NetworkHierarchyRootComponent).
+     * A network hierarchy is a collection of entities with one @NetworkHierarchyRootComponent at the top parent
```

Comment:
Is it true that there is only one NetworkHierarchyRootComponent? or can you have a parent with NetworkHierarchyRootComponent and grandparent with Root as well?  Example: Player prefab that jumps into a jeep prefab?
Hmm, I guess at edit time you should only ever have the one root but runtime that isn't the case?

Reply:
You can have multiple Roots within a hierarchy, anything but the top one will act as Children. There are unit tests in this CR that specifically test for it.